daffodil-lib/src/test/scala/org/apache/daffodil/util/TestCoroutines.scala

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
Highlight: Just adds back the missing unit tests for Coroutines library. Somehow these had gotten lost. This fixes DAFFODIL-2503